packages/sanity/src/core/store/_legacy/document/document-store.ts

```diff
@@ -94,6 +94,7 @@ export interface DocumentStore {
       publishedId: string,
       type: string,
       version?: string,
```

Comment:
Could we use this `version` argument instead of `displayedDocumentId` to determine what version to actually validate? E.g. if `version` was `'published' | 'draft' | '<releaseId>'`  then we'd be able to pick the right one for validating?